### ISSUE TITLE
not return the value after methods 

### ISSUE THREAD

**Issue by maartenv1** [tig-nl/postnl-magento2#62]
On several places the TIG_PosstNL does not follow the standard, it does not return the value after methods. An example here

vendor/tig/postnl-magento2/Plugin/Quote/ResetDeliveryDate.php

**original**

 ```
public function afterAddProduct(Quote $subject, $result)
    {
        // Since 1.4.2 the shipping duration can be configured on individual products.
        // So if deliveryDate is set and new quote items are added to the cart, it needs should be recalculated.
        if ($this->checkoutSession->getPostNLDeliveryDate()) {
            $this->checkoutSession->setPostNLDeliveryDate(false);
        }
    }
}
```

**The magento standard.**

   ```
 public function afterAddProduct(Quote $subject, $result)
    {
        // Since 1.4.2 the shipping duration can be configured on individual products.
        // So if deliveryDate is set and new quote items are added to the cart, it needs should be recalculated.
        if ($this->checkoutSession->getPostNLDeliveryDate()) {
            $this->checkoutSession->setPostNLDeliveryDate(false);
        }
		return $result;
    }
}

```

We encountered this problem when we could not checkout with onestepcheckout.com

**Comment by tig-jaspersmits** [comment]
Hello Maarten,

Thank you for providing us with this information. I will inform my colleague about this.
Once i have a response i will let you know.

Kind regards,

Jasper
TIG

**Comment by tig-jaspersmits** [comment]
Hi Maarten,

Just talked with my colleague about this and this was indeed a mistake in the release. This has been solved in version 1.4.3. which is available for download. Looking at the code you are probably running version 1.4.0.

Kind regards,

Jasper
TIG

**Comment by tig-jaspersmits** [comment]
Hi Maarten,

Another response from my side, it seems that this fix hasn't been pushed in version 1.4.3. yet. This is being updated as we speak and we will release a version 1.4.4. as soon as it is ready with these this fixes.

Kind regards,

Jasper
TIG

**Comment by tig-jaspersmits** [comment]
Hi Maarten,

Version 1.4.4 has just been released where this is fixed.
